tools/build_defs/pkg/pkg.bzl

```diff
@@ -200,6 +211,7 @@ _real_pkg_tar = rule(
         "empty_files": attr.string_list(),
         "include_runfiles": attr.bool(default = False, mandatory = False),
         "empty_dirs": attr.string_list(),
+        "remap_paths": attr.string_dict(),
```

Comment:
can you add to README.md.
Even better if we move the docs to the attrs and generate with Stardoc, but that should be a different CL.

Reply:
Done!